scripts/getBalances/tokenBalances.ts

```diff
@@ -0,0 +1,98 @@
+import * as fs from "fs";
+import { createDfuseClient } from "@dfuse/client";
+import dotenv from "dotenv";
+import * as csv from "fast-csv";
+
+dotenv.config();
+
+global.fetch = require("node-fetch");
+global.WebSocket = require("ws");
+
+const { DFUSE_GRAPHQL_URL, DFUSE_KEY, TOKEN_CONTRACT } = process.env;
+
+const client = createDfuseClient({
+  apiKey: DFUSE_KEY,
+  network: DFUSE_GRAPHQL_URL,
+});
+
+const operation = `query($contract: String!, $symbol:String!, $limit: Uint32, $cursor:String, $opts: [ACCOUNT_BALANCE_OPTION!]) {
+    tokenBalances(contract: $contract, symbol: $symbol,limit: $limit, cursor: $cursor, options: $opts) {
+      blockRef {
+        id
+        number
+      }
+      pageInfo {
+        startCursor
+        endCursor
+      }
+      edges {
+        node {
+          account
+          contract
+          symbol
+          precision
+          balance
+        }
+      }
+    }
+  }`;
+
+const { argv } = process;
+
+const variableConst = {
+  contract: TOKEN_CONTRACT,
+  symbol: argv[2],
```

Comment:
we should accept an additional optional 'contract' command param. If the token has a different contract you should be able to pass it in. If no contract arg is provided, use DEFAULT_TOKEN_CONTRACT
e.g. tokenBalances.ts ORE
e.g. tokenBalances.ts MYTOK mycontractxx